docs/en/latest/admin-api.md

```diff
@@ -1349,3 +1349,65 @@ Stream Route resource request address:  /apisix/admin/stream_routes/{id}
 | protocol.conf | False    | Configuration | Protocol-specific configuration.                             |                    |
 
 To learn more about filtering in stream proxies, check [this](./stream-proxy.md#more-route-match-options) document.
+
+## kms
+
+kms means `Secrets Management`, which could use any secret manager supported, e.g. `vault`.
+
+### kms API
+
+kms resource request address:: /apisix/admin/kms/{secretmanager}/{id}
```

Comment:
```suggestion
kms resource request address: /apisix/admin/kms/{secretmanager}/{id}
```

Reply:
Also, should we add a ``` around API?

Reply:
Let's not modify it now, let us modify it together in the next version.